### PR TITLE
Include filename in the error message

### PR DESCRIPTION
**Pull request by ElDeveloper**
Brief summary of the Pull Request, including any issues it may fix using the GitHub closing syntax:

https://help.github.com/articles/closing-issues-using-keywords/

Also, include any co-authors or contributors using the GitHub coauthor tag:

https://help.github.com/articles/creating-a-commit-with-multiple-authors/

---

Include any questions for reviewers, screenshots, sample outputs, etc.

### CODE REVIEW

**Review by thermokarst — APPROVED** [review]
Great addition, @ElDeveloper!

I pushed up a few minor tweaks (mostly just for eslint's sake): 67e8631

Feel free to revert/edit/etc - thanks!

**Comment by thermokarst** [comment]
:( Getting different eslint failures on travis vs locally. Boo.

**Comment by ElDeveloper** [comment]
Thanks @thermokarst should be fixed now.

**Comment by Oddant1** [comment]
Looks like Travis is still being a pain with the lint errors

**Comment by ElDeveloper** [comment]
@Oddant1 @thermokarst, looks like I had a very old version of eslint installed. Should be good now. I added a bunch of fixes but those were all generated by eslint itself.

File: app/pages/Home/component.jsx [hunk 1]
```diff
@@ -11,8 +11,9 @@ export default ({ submitFile, submitURL, redirectTo }) => (
         <Row>
             <Col md={12}>
                 <p style={{ fontSize: '18px' }}>
-                    This interface can view .qza and .qzv files directly in your browser without
-                    uploading to a server. <Link to="/about">Click here</Link> to learn more.
+                    EXAMPLE This interface can view .qza and .qzv files
```
Comment (thermokarst) [inline app/pages/Home/component.jsx]: What is the "EXAMPLE" for here? Maybe its just something leftover from development?

**Review by thermokarst — COMMENTED** [review]
LGTM, thanks @ElDeveloper! That is a bunch of annoying whitespace changes - thanks for dealing with that. I have one minor question/comment - inline. Thanks!

File: app/pages/Home/component.jsx (hunk 1 above)
Comment (ElDeveloper) [reply app/pages/Home/component.jsx]: Thanks so much, you are right, this was leftover and should have not been committed.

**Pull request merged**